### PR TITLE
fix test/requires being ignored when --no-copy-test-source-files

### PR DESCRIPTION
**Pull request by msarahan**
addresses https://github.com/vmtk/vmtk/pull/313#issuecomment-406146264

CC @jcfr

### CODE REVIEW

File: tests/test_api_build.py [hunk 1]
```diff
@@ -1185,6 +1185,14 @@ def test_copy_test_source_files(testing_config):
             assert not copy, "'info/test/test_files_folder/text.txt' not found in tar.bz2 but copying test source files. File list: %r" % files
 
 
+def test_copy_test_source_files_deps(testing_config):
+    recipe = os.path.join(metadata_dir, '_test_test_source_files')
+    for copy in (False, True):
+        testing_config.copy_test_source_files = copy
+        # test is that pytest is a dep either way.  Builds will fail if it's not.
```
Comment (jcfr) [inline tests/test_api_build.py]: `test is that pytest` -> `test that pytest`

Comment (msarahan) [reply tests/test_api_build.py]: Thanks.  I think this is correct wording - I'm saying what the test is.  Anyway, I'm both too lazy and not willing to do another CI cycle to change this.

**Comment by msarahan** [comment]
Test fails are unrelated.  Merging.

**Comment by jcfr** [comment]
Thanks for addressing the issue so quickly :+1:

**Pull request merged**